src/python/pants/backend/python/lint/docformatter/subsystem.py

```diff
@@ -29,32 +28,17 @@ class Docformatter(PythonToolBase):
     default_lockfile_path = "src/python/pants/backend/python/lint/docformatter/lockfile.txt"
     default_lockfile_url = git_url(default_lockfile_path)
 
-    @classmethod
-    def register_options(cls, register):
-        super().register_options(register)
-        register(
-            "--skip",
-            type=bool,
-            default=False,
-            help=f"Don't use docformatter when running `{bin_name()} fmt` and `{bin_name()} lint`.",
-        )
-        register(
-            "--args",
-            type=list,
-            member_type=shell_str,
-            help=(
-                "Arguments to pass directly to docformatter, e.g. "
-                f'`--{cls.options_scope}-args="--wrap-summaries=100 --pre-summary-newline"`.'
-            ),
-        )
-
-    @property
-    def skip(self) -> bool:
-        return cast(bool, self.options.skip)
-
-    @property
-    def args(self) -> Tuple[str, ...]:
-        return tuple(self.options.args)
+    skip = BoolOption(
+        "--skip",
+        default=False,
+        help=f"Don't use docformatter when running `{bin_name()} fmt` and `{bin_name()} lint`.",
+    )
+    args = ArgsListOption(
+        help=lambda cls: (
+            "Arguments to pass directly to docformatter, e.g. "
+            f'`--{cls.options_scope}-args="--wrap-summaries=100 --pre-summary-newline"`.'
+        ),
+    )
```

Comment:
Is it necessary to use this when we don't expect subclassing? That's fine if the argument is consistency.

Reply:
No it isn't. you can just use `options_scope`. I'll fix those in a follow-up because more than this PR has that quirk.